apps/web/src/components/ui/RichTextEditor.tsx

```
@@ -0,0 +1,325 @@
+  const isActive = (name: string) => {
+    txVersion(); // Subscribe to transaction changes for Solid reactivity
+    return editor()?.isActive(name) ?? false;
+  };
+
+  return (
+    <div class={`flex flex-col gap-1.5 ${props.class || ""}`}>
+      <Show when={props.label}>
+        <label id={labelId} class="font-mono text-xs uppercase tracking-wider text-stone">
+          {props.label}
+        </label>
+      </Show>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)